src/main/java/by/exadel/internship/service/FormService.java

```diff
@@ -67,13 +70,46 @@ private void uploadFile(MultipartFile file, UUID uuid) {
                             (new File(filePath + File.separator + uuid +
                                     File.separator + file.getOriginalFilename())));
             stream.write(bytes);
-            log.info("Success to upload file, form id: "+ uuid);
+            log.info("Success to upload file, form id: " + uuid);
             stream.close();
         } catch (IOException e) {
             log.error(e.getMessage());
         }
     }
 
+    private void updateFilePath(Form createdForm, MultipartFile file) {
```

Comment:
After synchronization with dev this method was removed. Check it usage and remove if it's not used

Reply:
Yes, we are not using this method now